tests/test_units.py

```
@@ -519,3 +522,68 @@ def test_fork_handler_replaces_held_lock(self):
+        """Only quantities can be data sizes."""
+        assert not is_data_size(value)
+
+    def test_millibarn_is_not_megabytes(self):
+        """`mb` is millibarn (an area) in pint; only `MB` is megabytes."""
+        assert not is_data_size(dc.units.mb)
+        assert is_data_size(dc.units.MB)
+
+    def test_undefined_byte_alias_raises(self):
+        """`KB` is not a pint unit; the kilobyte spelling is `kB`."""
+        with pytest.raises(pint.UndefinedUnitError):
+            dc.units.KB
```

Ruff (0.16.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)